packages/portals/frontend/sirius-components-portals/src/representations/PortalRepresentation.tsx

```diff
@@ -152,7 +152,8 @@ export const PortalRepresentation = ({
         w: 10,
         h: 20,
         static: true,
-      }}>
+      }}
+      data-testid="portal-drop-area">
```

Comment:
This data testid is already present, it prevents React from running the application since you can't have the same attribute twice.